actions/docs/guides/02-chat-files.md

```diff
@@ -0,0 +1,83 @@
+## Working with Files in Chat Actions
```

Comment:
We should also add a section saying why this is needed (something along the lines that when running in the cloud, accessing anything in the local file system isn't possible and these APIs should be used to access anything that needs to be thread-scoped).